charts/spire/charts/spire-server/README.md

```diff
@@ -60,6 +61,7 @@ A Helm chart to install the SPIRE server.
 | imagePullSecrets | list | `[]` |  |
 | initContainers | list | `[]` |  |
 | jwtIssuer | string | `"oidc-discovery.example.org"` |  |
+| jwtSvidTTL | string | `"1h"` |  |
```

Comment:
Should we have the default in front like the spire server does? They would be next to each other in the list if so.

Reply:
Thought about that, but then started thinking there isn't some setting as a non-default. Both are fine for me. Just found the default thing a bit weird from Helm perspective, but I might be missing some reasoning behind that default.

Reply:
+1 for keeping the option name the same I think?

default is because the TTL can be overridden at the per entry level in the spire db. So its the default TTL when no override is set.